### ISSUE TITLE
Romeo keymap wrong keycode for Enter switch?

### ISSUE THREAD

**Issue by BenRoe** [qmk/qmk_firmware#10084]
Maybe i am wrong, but shouldn't the most right switch in the second row from top, be a `KC_ENT` and not a `KC_QUOT`?
@coseyfannitutti 
https://github.com/qmk/qmk_firmware/blob/186b1b2ba2550d9a7a864a241e8f9503c37f188b/keyboards/coseyfannitutti/romeo/keymaps/default/keymap.c#L22

![](https://images.squarespace-cdn.com/content/v1/5c533d33348cd92b886e544d/1593746778951-MAZ2HB3E0HZBQS9OYI9X/ke17ZwdGBToddI8pDm48kK60W-ob1oA2Fm-j4E_9NQB7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0kD6Ec8Uq9YczfrzwR7e2Mh5VMMOxnTbph8FXiclivDQnof69TlCeE0rAhj6HUpXkw/romeo-black.jpeg)

![](https://images.squarespace-cdn.com/content/v1/5c533d33348cd92b886e544d/1593748095502-ODHBDNA2SL4K9JKM6JIJ/ke17ZwdGBToddI8pDm48kK60W-ob1oA2Fm-j4E_9NQB7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0kD6Ec8Uq9YczfrzwR7e2Mh5VMMOxnTbph8FXiclivDQnof69TlCeE0rAhj6HUpXkw/romeo-white.jpg?format=1500w)

**Comment by coseyfannitutti** [comment]
This was an intentional choice on my part, and maybe I should have been more clear with my reasoning on the project's page. When I was testing the layout, I found myself constantly hitting enter and cutting my sentences off when chatting in Discord, for example, and trying to hit apostrophe. I found that placing enter on layer 1 made striking enter more intentional.

Because it's a 40% with a mostly standard staggered layout, I expected a large percentage of users to be new to 40% and/or unconventional layouts. I think it could help touch typists not used to a compact layout to be able to mostly type naturally without accidentally submitting forms or chat messages.

I figured if anyone didn't like it, they could easily rebind it in VIA.
